assets/templates/kubernetes.pod/gotext.tpl

```diff
@@ -0,0 +1,18 @@
+{{- $period := generate "metricset.period" -}}
+{{- $timestamp := generate "timestamp" -}}
+{{- $fulltimestamp := $timestamp.Format "2006-01-02T15:04:05.999999Z07:00" -}}
+{{- $resttime := split ":" $fulltimestamp -}}
+{{- $picktimedate := generate "timedate" -}}
+{{- $timehour := generate "timehour" -}}
+{{- $agentId := generate "agent.id" -}}
+{{- $agentVersion := generate "agent.version" -}}
+{{- $agentName := generate "agent.name" -}}
+{{- $agentEphemeralid := generate "agent.ephemeral_id" -}}
+{{- $rxbytes := generate "container.network.ingress.bytes" -}}
+{{- $txbytes := generate "container.network.egress.bytes" -}}
+{{- $uId := uuidv4 -}}
+{{- $pod_uId := uuidv4 -}}
+{{- $suffix := split "-" $uId -}}
+{{- $offset := generate "Offset" | int -}}
```

Comment:
nit: `Offset` still casted to `int`

Reply:
fixed in [1f0eec3](https://github.com/elastic/elastic-integration-corpus-generator-tool/pull/88/commits/1f0eec3b7cb21974560340daa4edae971a77465d)